src/python/espressomd/MDA_ESP/__init__.py

```diff
@@ -30,7 +30,7 @@
 <Universe with 1 atoms>
 
 """
-import cStringIO
+from six import StringIO
```

Comment:
`six` is not part of the Python standard library.  I would prefer if the code wasn't introducing additional dependencies.

Reply:
There is already an import of six in `src/python/espressomd/io/writer/h5md.py` and MDAnalysis requires it anyway. If you prefer, I can replace this by a try/except block or a conditional import that check the version of Python.

Reply:
Also, if needed, [six](https://github.com/benjaminp/six)

> (...) is contained in only one Python file, so it can be easily copied into your project.

But it is on PyPI and does not carry further dependencies.

Reply:
We don't need to use six for this. What it does is trivial and you can do the same using something like:
```
try:
    import cStringIO as StringIO
except ImportError:
    from io import StringIO
```